dependency/test/jammy.Dockerfile

```diff
@@ -1,9 +1,11 @@
-FROM ubuntu:22.04
+FROM paketobuildpacks/build-jammy-full
```

Comment:
We're still installing a whole bunch of packages that are on the stack image (starting on line 10). Can we please pare this down?